CovertInfiltration/Config/XComGameData.ini

```diff
@@ -49,3 +49,116 @@ m_iMaxSoldiersOnMission=6
 [XComGame.X2AbilityTemplateManager]
 AbilityRetainsConcealmentVsInteractives="AdventDisguiseStats"
 AbilityRetainsConcealmentVsInteractives="HolographicDisguiseStats"
+
+[XComGame.X2SitRep_DefaultSitReps]
+; removed
+-SitReps=Surgical
+-SitReps=TheHorde
+-SitReps=LowProfile
+-SitReps=ShowOfForce
+-SitReps=PsionicStorm
+-SitReps=Savage
+-SitReps=AutomatedDefenses
+;added
++SitReps=InformationWar_CI
++SitReps=FamiliarTerrain_CI
++SitReps=PhysicalConditioning_CI
+;+SitReps=MentalReadiness_CI
++SitReps=LightningStrike_CI
++SitReps=AdventPatrols_CI 
+;+SitReps=GunneryEmplacements_CI
++SitReps=Phalanx_CI
++SitReps=Congregation_CI
++SitReps=ShoddyIntel_CI
++SitReps=WellRehearsed_CI
++SitReps=IntelligenceLeak_CI
++SitReps=VolunteerArmy_CI
++SitReps=DoubleAgent_CI
++SitReps=TacticalAnalysis_CI
++SitReps=AdventAirPatrols_CI
++SitReps=CommsJamming_CI
++SitReps=OpportuneMoment_CI
```

Comment:
There is no need to name **_CI** unless you are overriding a base game sitrep. Same with effects